### PR TITLE
Do not package example files

### PR DESCRIPTION
**Pull request by ChristophWurst**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -46,6 +46,7 @@ appstore:
 	--exclude=phpunit*xml \
 	--exclude=tests \
 	--exclude=vendor/bin \
+	--exclude=vendor/yubico/u2flib-server/examples \
```
Comment (LukasReschke) [inline Makefile]: I'd rather prefer to remove line 27. The stuff is already checked in in git anyways

Comment (ChristophWurst) [inline Makefile]: how about we do both, just to be sure not to package it in case I do a local composer install? :-)

Comment (ChristophWurst) [inline Makefile]: that line was not used, but I removed it

Comment (ChristophWurst) [inline Makefile]: ping :-)

Comment (LukasReschke) [inline Makefile]: 👍

**Pull request merged**